extensions/omni_schema/src/schema_ordering.sql

```diff
@@ -0,0 +1,255 @@
+    ) on commit drop;
+
+    -- conn for setting up table_fs in remote database
+    perform dblink_connect(remote_table_fs_name, conn_info);
+
+    perform dblink(remote_table_fs_name, 'create extension if not exists omni_vfs cascade');
+    -- create table_fs in remote database
+    select
+        (f).id
+    from
+        dblink(remote_table_fs_name,
+               format('select omni_vfs.table_fs(%L)', remote_table_fs_name)) as t(f omni_vfs.table_fs)
+    into remote_table_fs;
+
+    -- read all files recognised by omni_schema.languages and omni_schema.auxiliary_tools
+    for rec in
+        select
+            path || '/' || name                                           as name,
+            convert_from(omni_vfs.read(fs, path || '/' || name), 'UTF-8') as code,
+            language,
+            extension,
+            file_processor,
+            file_processor_extension,
+            processor,
+            processor_extension
+        from
+            omni_vfs.list_recursively(fs, path) files
+            left join omni_schema.languages on files.name like concat('%', languages.file_extension)
+            left join omni_schema.auxiliary_tools
+                      on files.name like concat(coalesce(auxiliary_tools.filename_stem, '%'), '.',
+                                                auxiliary_tools.filename_extension)
+        where
+            (auxiliary_tools.id is not null or
+             languages.id is not null) and
+            kind = 'file'
+        order by coalesce(auxiliary_tools.priority, languages.priority) desc, name
+        loop
+            -- create table_fs entry for the file along with it's data
+            perform dblink(remote_table_fs_name,
+                           format(
+                                   'insert into omni_vfs.table_fs_files(filesystem_id, filename, kind) values (%s, %L, ''file'')',
+                                   remote_table_fs.id, rec.name));
+            perform dblink(remote_table_fs_name,
+                           format(
+                                   'insert into omni_vfs.table_fs_file_data (file_id, data) values
+                                      (
+                                        omni_vfs.table_fs_file_id(omni_vfs.table_fs(%L), %L),
+                                        %L
+                                      )',
+                                   remote_table_fs_name, rec.name, convert_to(rec.code, 'utf-8')));
+
+            if rec.code ~ 'omni_schema\[\[ignore\]\]' then
+                -- Ignore the file
+                continue;
+            end if;
+            if rec.language = 'sql' then
+                -- process sql language files statement by statement
+                for current_statement in select
+                                             source
+                                         from
+                                             omni_sql.raw_statements(convert_from(omni_vfs.read(fs, rec.name), 'UTF-8')::cstring)
```

Comment:
where should i use the column and line number ? as separate columns in return table or use it as a prefix only for error message of a sql statement